wcc-contentful-graphql/lib/wcc/contentful/graphql/types.rb

```diff
@@ -29,17 +29,35 @@ module WCC::Contentful::Graphql::Types
       field :lon, !types.Float, hash_key: 'lon'
     end
 
-  AnyScalarInputType =
-    GraphQL::ScalarType.define do
```

Comment:
completely doing away with Scalars?

Reply:
I had this wrong - I thought I could define an 'Any' type and it would allow anything.  But what that really means is it only accepts values of the 'any' type.

So I couldn't do this:

```graphql
query myQuery($slug: String) {
  allPerson(filter: { slug: { eq: $slug } })
}
```
because `eq` there was expecting something of type 'Any' not of type 'String'.